addons/html_editor/static/src/main/align/align_plugin.js

```diff
@@ -35,11 +34,10 @@ export class AlignPlugin extends Plugin {
                 run: () => this.setAlignment("justify"),
             },
         ],
-        toolbar_groups: withSequence(29, { id: "alignment" }),
         toolbar_items: [
             {
                 id: "alignment",
-                groupId: "alignment",
+                groupId: "list_and_alignment",
```

Comment:
Can we rename this to something else? I suggest `layout`. What do you think? Do you have a better idea?

Reply:
I think "layout" is a good name.
Personally I found it surprising to have list and text aligment in the same group (other than looking good), and I even double checked it with Lionel.
But naming such group "layout" kinda makes it more OK.

Reply:
Hence the importance of finding good names! 😛